lib/shopify-cli/commands/create.rb

```diff
@@ -21,8 +21,12 @@ def self.extended_help
           {{bold:Subcommands:}}
 
             {{cyan:project}}: Creates an app based on type selected.
-              Usage: {{command:#{ShopifyCli::TOOL_NAME} create project <appname>}} or
-                     {{command:#{ShopifyCli::TOOL_NAME} create project <appname> --type=TYPE}]
+              Usage: {{command:#{ShopifyCli::TOOL_NAME} create project <appname>}} or any combination of
```

Comment:
Do we need to repeat the command over and over? Does that send the right message? (It is possible everyone already had this discussion, sorry)

Reply:
Very good question! We haven’t really discussed. It does seem redundant for sure

Reply:
I don't think we discussed before, but agreed, I think we can do this without repeating the command each time. How about:

```
Usage: 
  {command}

Options:
  --type      {definition}
  --title     {definition}
...etc
```

Reply:
I like this way more!